### PR TITLE
Add UBP icon

### PR DESCRIPTION
**Pull request by candrasaputra**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Review by tauthomas01 — APPROVED** [review]
Looks great. Thank you for contributing

I can't merge the PR due to Github's Actions CI not responding. It seems like Github is experimenting some issues so we'll keep you posted.

https://www.githubstatus.com/incidents/zbpwygxwb3gw?utm_ts=1621149460

**Comment by candrasaputra** [comment]
Noted, thank you @tauthomas01

**Comment by candrasaputra** [comment]
Hi @tauthomas01 any update regarding automated testing? it's look stuck.

File: db/payment_icons.yml [hunk 1]
```diff
@@ -956,6 +956,6 @@
   group: credit_cards
 -
   name: ubp
-  label: UBP
+  label: UBP X
```
Comment (larouxn) [inline db/payment_icons.yml]: Was this intentional?

**Comment by larouxn** [comment]
> Hi @tauthomas01 any update regarding automated testing? it's look stuck.

We're still looking into it. Have reached out to some people. Rather peculiar issue as nearly identical GitHub Actions based CI configurations in other repos are running fine. (i.e. [this repo](https://github.com/activemerchant/payment_icons/blob/62933fd51beb625f39de59e597910169e6cdb9ba/.github/workflows/ruby-ci.yml) vs [another repo](https://github.com/larouxn/carrotgram/blob/c5d58bd1a0d36a4d13e4519f8765cfd61baaf00b/.github/workflows/ruby-ci.yml))

**Comment by tauthomas01** [comment]
Seems to work now. I'll go ahead and merge.

**Comment by candrasaputra** [comment]
@tauthomas01 cool.. thank you.

could i ask, any ETA for deployment?

**Comment by larouxn** [comment]
> @tauthomas01 cool.. thank you.
> 
> could i ask, any ETA for deployment?

@candrasaputra, looks like maybe next week.

**Comment by larouxn** [comment]
👋 @candrasaputra, the UBP icon should be available everywhere now.

**Pull request merged**